### ISSUE TITLE
Terraform documentation web UI/UX little bit awkward

### ISSUE THREAD

**Issue by markotitel** [hashicorp/terraform-website#1973]
Hi,
I got advice on Reddit to create an issue about documentation part of the website.

For example when I read resource parameters I want to COPY them but that is impossibile right now.

Could you helpout? = )

Here is related Reddit post.
https://www.reddit.com/r/hashicorp/comments/qwozlu/can_terraform_documentation_be_more_ui_friendly/

**Comment by sudomateo** [comment]
@jescalan is this something that would fall under your domain?

I'm in agreement that the current user experience is subpar for copying provider attributes. However, I understand that there is value in anchor linking to a specific attribute. I'm wondering if there is a compromise we can make here.

Here's what a given provider attribute currently looks like. It's difficult to highlight without clicking, and the act of clicking will send the user to the anchor link and move the page content around.

![image](https://user-images.githubusercontent.com/24284972/142662403-287ff75b-452e-4c5e-8ebf-155550b8a965.png)

**Comment by kingletas** [comment]
@sudomateo @markotitel I too agree the UI could be better. The way I have been handling it today is that I copy the parameter from the navigation bar. Not ideal.

**Comment by jescalan** [comment]
Yeah this is a good call. Auto-linking parameter names like this in HashiCorp docs has been around for at least 5 years at this point, and as you pointed out @sudomateo, I don't think we can drop the functionality. However, we have discussed and have an internal ticket in adjusting the anchor link style to match the way it looks on headlines (when you hover the word, a permalink icon appears to the left) rather than this method where the word itself is linked.

I will give this one a boost internally and we'll see if we can make it happen!
